webete/webete.py

```diff
@@ -10,6 +10,8 @@
 import xdis.load
 import uncompyle6.semantics.pysource
 
+from requests.auth import HTTPBasicAuth
```

Comment:
Is this ever used? Anyway refrain from importing stuff into the global namespace, especially if you use them only once.
Also, not a show stopper but usually I put my imports like this:
```
import python_standard_library_stuff

import external_libs

import stuff from the current project
```